src/components/ProductRow.jsx

```diff
@@ -0,0 +1,19 @@
+function ProductRow(props) {
+  return (
+    <div>
```

Comment:
I would use a fragment here so you can keep the same structure for the table

```suggestion
    <>
```